src/flp.rs

```diff
@@ -1,12 +1,13 @@
 // SPDX-License-Identifier: MPL-2.0
 
 //! Implementation of the generic Fully Linear Proof (FLP) system specified in
-//! [[draft-irtf-cfrg-vdaf-01]]. This is the main building block of [`Prio3`](crate::vdaf::prio3).
+//! [[draft-irtf-cfrg-vdaf-03]]. This is the main building block of [`Prio3`](crate::vdaf::prio3).
 //!
 //! The FLP is derived for any implementation of the [`Type`] trait. Such an implementation
 //! specifies a validity circuit that defines the set of valid measurements, as well as the finite
 //! field in which the validity circuit is evaluated. It also determines how raw measurements are
-//! encoded as inputs to the validity circuit.
+//! encoded as inputs to the validity circuit, and how aggregates are decoded from sums of
```

Comment:
nit: "sum" isn't the right term for all VDAFs. At the risk of making this less clear, could we use something more generic? "combinations of measurements"? Or 
```suggestion
//! encoded as inputs to the validity circuit, and how aggregation results are decoded from aggregations of measurements
```

Reply:
This module is restricted to just Prio3, so there will be a sum of some kind. Maybe this would be more precise?
```suggestion
//! encoded as inputs to the validity circuit, and how aggregation results are decoded from sums of encoded
```

Reply:
Good point, we do literally sum the aggregate shares together in the Prio case. Let's leave the text as is.